source/ui/dataman/AMScanThumbnailGridViewItemDelegate.h

```diff
@@ -0,0 +1,57 @@
+#ifndef AMSCANTHUMBNAILGRIDVIEWITEMDELEGATE_H
+#define AMSCANTHUMBNAILGRIDVIEWITEMDELEGATE_H
+
+#include <QItemDelegate>
+#include <QPainter>
+
+/**
+ * Delegate class for painting the items within an AMScanThumbnailGridView
+ */
+class AMScanThumbnailGridViewItemDelegate : public QItemDelegate
+{
+	Q_OBJECT
+public:
+	/**
+	 * Creates an instance of an AMScanThumbnailGridViewItemDelegate
+	 * @param parent
+	 */
+	explicit AMScanThumbnailGridViewItemDelegate(QObject *parent = 0);
+
+	/// Paints the data associated with the provided index according to options stored in the given
```

Comment:
With there being no resources owned by the delegate I just left it up to the default destructor. I can add an explicit destructor in here if required.